src/erlcloud_ec2.erl

```diff
@@ -1441,27 +1538,131 @@ extract_block_device_mapping_status(Node) ->
      {delete_on_termination, get_bool("ebs/deleteOnTermination", Node)}
     ].
 
-%%
-%%
--spec(describe_instance_status/1 :: (string()) -> proplist()).
-describe_instance_status(InstanceID) ->
-    describe_instance_status([{"InstanceId", InstanceID}], [], default_config()).
+-type instance_id() :: string().
+-type ec2_param_list() :: [{string(),string()}].
 
--spec(describe_instance_status/2 :: (proplist(), filter_list()) -> proplist()).
-describe_instance_status(Params, Filter) ->
-    describe_instance_status(Params, Filter, default_config()).
+-define(SPOT_INSTANCE_STATUS_MR_MIN, 5).
+-define(SPOT_INSTANCE_STATUS_MR_MAX, 1000).
 
--spec(describe_instance_status/3 :: (proplist(), filter_list(), aws_config()) -> proplist()).
-describe_instance_status(Params, Filter, Config) ->
+-spec describe_instance_status() -> ec2_normal_describe_return().
+describe_instance_status() ->
```

Comment:
There was no `describe_instance_status/0` before? What does it do? My guess it will describe statuses of all instances? So what `describe_instance_status_all/...` does?